contrib/blueflood-docker/docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 blueflood:
-  build: .
+  image: goru97/blueflood
```

Comment:
@usnavi @shintasmith had the same concern in https://github.com/rackerlabs/blueflood/pull/696. I reached out to #rackspace #respose channels on IRC, perused through one wiki and also e-mailed dev and devops list but nobody got the credentials for rackerlabs docker-hub account. usernames are part of the docker images if they are hosted on your docker hub account. They are not required if they are hosted by docker official account e.g. cassandra, elasticsearch.